src/renderers/widget.ts

```diff
@@ -272,7 +272,7 @@ class RenderedJavascript extends Widget {
     super();
     let s = document.createElement('script');
     s.type = options.mimetype;
-    s.textContent = options.source;
+    s.textContent = <string>options.source;
```

Comment:
The recommended way to cast in Typescript is the `as` keyword (brackets are an old relic): `options.source as string`.